charts/kserve/templates/configmap.yaml

```diff
@@ -200,6 +200,11 @@ data:
   transformers: |-
     {
     }
+  metricsAggregator: |-
```

Comment:
@yuzisun hey do you happen to know if I really need this duplicated like four times? I just looked for where the config/configmap/inferenceservice.yaml values were and found them in this charts directory, not sure if I need it though bc when I tested it worked but maybe im missing something?

Reply:
We need this for helm chart installation, KServe currently supports two installation methods: helm or kustomize.

Reply:
ok got it, hope I have it in the right places lol